### ISSUE TITLE
What happens if I finetune with my dataset in your model using only one video?

### ISSUE THREAD

**Issue by justinday123** [TIGER-AI-Lab/ConsistI2V#28]
What happens if I finetune with my dataset in your model using only one video? Does result looks well?

**Comment by wenhuchen** [comment]
You can try to do some shuffling and transformation of this particular video. It won't crash and it would overfit to this video and do some personalization. See https://arxiv.org/abs/2208.12242

**Comment by justinday123** [comment]
Thanks. You mean transforming or  shuffling all frames?

Input: transformed or shuffled frame, output: ground truth video.

Is this process right?

**Comment by wenhuchen** [comment]
No, just train on different transformation to maintain diversity.
